### DISCUSSION TITLE
Modified scripts to work in Addigy

### DISCUSSION THREAD

**Discussion by CentralizedServices** [Macjutsu/super#245] [Show and tell]
I modified the following scripts for the Super Friends folder to work as custom facts in Addigy. I added extra checks to catch errors that Addigy threw and shortened the output to make them compatible with the standard view. I also changed some code for those dates to show the time remaining before the next launch, not as a date but as days and hours.

Super OS Downloads
Super Focus Deferred
Super Installed  String
Super Next Launch Date
Super next launch time
Super Soft Deferred Count

Would you like copies of what I have? If so, how would you like me to get them to you?

Thanks

**Comment by Macjutsu** [comment]
Thank you! When I update the "Super-Friends" folder structure in the next minor release... we can work out an Addigy folder for theses scripts.
